deploy/olm-catalog/service-assurance-operator/0.1.1/service-assurance-operator.v0.1.1.clusterserviceversion.yaml

```diff
@@ -12,22 +12,23 @@ metadata:
           },
           "spec": {
             "eventsEnabled": false,
+            "highAvailabilityEnabled": false,
             "metricsEnabled": true
           }
         }
       ]
     capabilities: Basic Install
     categories: Monitoring
     certified: "false"
-    containerImage: quay.io/redhat-service-assurance/service-assurance-operator.0.1.0
-    createdAt: "2019-11-13T17:16:00Z"
+    containerImage: quay.io/leifmadsen/service-assurance-operator
+    createdAt: "2020-01-10T21:30:00Z"
     description: Service Assurance Framework. Umbrella Operator for instantiating
       the required dependencies and configuration of various components to build a
       Service Assurance platform for telco grade monitoring.
     repository: https://github.com/redhat-service-assurance/service-assurance-operator
     support: Red Hat, Inc.
-  name: service-assurance-operator.v0.1.0
-  namespace: placeholder
+  name: service-assurance-operator.v0.1.1
+  namespace: sa-telemetry
```

Comment:
This doesn't seem to affect anything, but is nice to have when you're trying to import the CSV directly for testing.